epics-release.py

```diff
@@ -222,10 +222,10 @@ def ValidateArgs( repo, packageSpec, opt ):
     ( opt, args ) = parser.parse_args()
 
     if not opt.release:
-        raise ValidateError, ( "Release tag not specified!" )
+        raise ValidateError(( "Release tag not specified!" ))
```

Comment:
The double parens don't cause any issues but seem out of place. This comes up a few times in this file and in other files, alongside other strange whitespace decisions.
```suggestion
        raise ValidateError("Release tag not specified!")
```

Reply:
Might give this a formatting pass in a future PR.